ansible/roles/ansible-datamover-api/defaults/main.yml

```diff
@@ -1,11 +1,13 @@
 # Database vars
-dmapi_db_setup_host: "{{ ('galera_all' in groups) | ternary(groups['galera_all'][0], 'localhost') }}"
+dmapi_db_setup_host_victoria: "{{ openstack_db_setup_host | default('localhost') }}"
+dmapi_db_setup_host_ussuri: "{{ ('galera_all' in groups) | ternary(groups['galera_all'][0], 'localhost') }}"
```

Comment:
dont create the variable names with openstack versions.
create common variables, but populate the values as per openstack version.
The same applies to the interpreter variable as well.

Reply:
changes done